src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Security/DoNotCallDangerousMethodsInDeserialization.cs

```diff
@@ -33,7 +33,7 @@ public sealed class DoNotCallDangerousMethodsInDeserialization : DiagnosticAnaly
 
         private ImmutableArray<(string, string[])> DangerousCallable = ImmutableArray.Create<(string, string[])>
             (
-                (WellKnownTypeNames.SystemIOFileFullName, new[] { "WriteAllBytes", "WriteAllLines", "WriteAllText", "Copy", "Move", "AppendAllLines", "AppendAllText", "AppendText", "Delete" }),
+                (WellKnownTypeNames.SystemIOFile, new[] { "WriteAllBytes", "WriteAllLines", "WriteAllText", "Copy", "Move", "AppendAllLines", "AppendAllText", "AppendText", "Delete" }),
```

Comment:
Can we retarget this PR to 2.9.x branch, given the changes are only in security related rules? Security team still works out of 2.9.x, not master, so this will help them avoid merge conflicts. I think it might be possible to just edit the base branch in the github PR UI itself...

Reply:
Okay no problem. I'll tey retargeting first, and if it's not possible, I will submit a new PR.